modules/dns/dns.rb

```diff
@@ -1,20 +1 @@
 require 'dns/dns_plugin'
-
-module Proxy::Dns
```

Comment:
I would very much like to do that, as otherwise we'll have weird dependency graph -- some things are going to depend on stuff in `dns_common`, others in `dns`. It wouldn't cause issues with dependency loading, but might cause confusion with locating of common dependencies (or require explanation why things are this way). I'll do a pr for route53, I assume @ekohl is ok doing the changes for powerdns?